src/main/java/com/metamx/collections/bitmap/WrappedImmutableConciseBitmap.java

```diff
@@ -52,7 +53,10 @@ public ImmutableConciseSet getBitmap()
   @Override
   public boolean get(int value)
   {
-    return bitmap.get(value) > 0;
+    final ConciseSet conciseSet = new ConciseSet();
+    conciseSet.add(value);
+    final ImmutableConciseSet immutableConciseSet = ImmutableConciseSet.newImmutableFromMutable(conciseSet);
+    return ImmutableConciseSet.intersection(this.bitmap, immutableConciseSet).size() != 0;
```

Comment:
Since the class implements an iterator: https://github.com/metamx/bytebuffer-collections/blob/master/src/main/java/com/metamx/collections/bitmap/WrappedImmutableConciseBitmap.java#L77 , I think iterating over the current bitmap until meeting the requested int or the first int greater than the searched one should be more faster than creating a new bitmap and performing its inetrsection with the current one.